### DISCUSSION TITLE
Aesara Installation Error in Windows

### DISCUSSION THREAD

**Discussion by purna135** [aesara-devs/aesara#877] [General]
To configure aesara environment in my Windows pc , I used https://aesara.readthedocs.io/en/latest/dev_start_guide.html
However, when I run `pip install -r requirements.txt`, I get the following error:
```
Using cached jax-0.3.4.tar.gz (924 kB)
  Preparing metadata (setup.py) ... done

ERROR: Could not find a version that satisfies the requirement jaxlib (from versions: none)
ERROR: No matching distribution found for jaxlib
```
What is the best way to fix this problem?

**Comment by brandonwillard** [comment]
Can you list all the steps to reproduce this?

**Reply by brandonwillard** [reply]
Actually, you may want to check https://github.com/google/jax/issues/438 first.

**Reply by purna135** [reply]
Thank you @brandonwillard, got it now
